services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls(cinema_halls: int) -> list[int]:
```

Comment:
The type hint for the return value of `get_cinema_halls` is incorrect. The function returns a `QuerySet`, not a list of integers. Change the return type hint to `QuerySet[CinemaHall]`.